Apps/W1/HybridGP/test/src/GPItemTests.codeunit.al

```diff
@@ -46,26 +47,38 @@ codeunit 139662 "GP Item Tests"
             Migrate(GPItem);
         until GPItem.Next() = 0;
 
-        // [THEN] A Item is created for all staging table entries
-        Assert.RecordCount(Item, GPItem.Count());
-        Assert.AreEqual(GPItem.Count(), HelperFunctions.GetNumberOfItems(), 'Wrong number of Items calculated');
+        HelperFunctions.CreatePostMigrationData();
+
+        // [THEN] An Item is created for all configured staging table entries
+        Assert.RecordCount(Item, 7);
+        Assert.AreEqual(Item.Count(), HelperFunctions.GetNumberOfItems(), 'Wrong number of Items calculated');
 
         // [THEN] Items are created with correct settings
-        GPItem.FindSet();
-        Item.FindSet();
-        repeat
-            Assert.AreEqual(GPItem.No, Item."No.", 'Item No. not set');
-            Assert.AreEqual(0.00, Item."Unit Price", 'Unit Price set');
-            Assert.AreEqual(GPItem.CurrentCost, Item."Unit Cost", 'Unit Cost not set');
-            Assert.AreEqual(GPItem.StandardCost, Item."Standard Cost", 'Standard Cost not set');
-            Assert.AreEqual(GPItem.ShipWeight, Item."Net Weight", 'Net Weight not set');
-            Assert.AreEqual(GPItem.BaseUnitOfMeasure, Item."Base Unit of Measure", 'Base Unit of Measure not set');
-            Assert.AreEqual(GPItem.Description, Item.Description, 'Description not set.');
-            Assert.AreEqual(GPItem.ShortName, Item."Description 2", 'Description2 not set.');
-            Assert.AreEqual(GPItem.SearchDescription, Item."Search Description", 'Search Description not set.');
-            Assert.AreEqual(GPItem.PurchUnitOfMeasure, Item."Purch. Unit of Measure", 'Purch. Unit of Measure not set.');
-            GPItem.Next();
-        until Item.Next() = 0;
+        GPItem.Get('1 1/2\"SASH BRSH');
```

Comment:
Similar comment here, lets introduce a variable at least if it is not possible to calculate the data